### ISSUE TITLE
How to delete planb-cassandra

### ISSUE THREAD

**Issue by MALPI** [zalando/planb-cassandra#27]
It would be nice to have some documentation which steps need to be done in order to delete planb cassandra again.

**Comment by MALPI** [comment]
I deleted now the EBS-Volumes, EC2 isntances and security group, is this sufficient?

**Comment by a1exsh** [comment]
Probably the IAM role also.

**Comment by MALPI** [comment]
I cannot find any IAM role which could be related to planb actually

**Comment by a1exsh** [comment]
You're right, I confused it with something else.  Then there's nothing more to remove (except for those elastic IPs, if you didn't release them at once).

There is planned work to extend the cluster control script to provide commands to delete the cluster.

**Comment by MALPI** [comment]
Just another note, I also had to remove the Elastic IPS.

**Comment by LappleApple** [comment]
Hey @a1exsh + @MALPI, can we close this issue? It dates back to May 2016.

**Comment by a1exsh** [comment]
We are actually right on it.  We have a prototype branch to provide more subcommands to list, update, delete clusters, etc.
